tools/c7n_mailer/README.md

```diff
@@ -191,6 +191,57 @@ Slack delivery can also be set via a resource's tag name. For example, using "sl
 
 Delivery via tag has been tested with webhooks but should support all delivery methods.
 
+### Jira:
+
+The Custodian mailer supports Jira as a separate notification mechanism for the SQS transport method. To enable Jira integration, you must specify below fields in the mailer config file.
+
+```yaml
+jira_url: https://your-org.atlassian.net/
+jira_username: YOUR_JIRA_USERNAME
+jira_token: YOUR_JIRA_TOKEN
+# optional, the dict to set custom fields for each Jira project when needed
+jira_custom_fields:
+  # Set fields for all projects by using 'DEFAULT' section
+  DEFAULT:
+    priority:
+      name: Medium
+  # Add special fields for a Jira project
+  MY_PROJECT:
+    customfield_10059: value_for_the_field
+  # Remove special fields for a Jira project with keyword 'cannot-be-set'
+  MY_ANOTHER_PROJECT:
+    priority: cannot-be-set
+```
+
+To enable Jira integration, several unique fields are evaluated in the policy, as shown in the below example:
+
+```yaml
+actions:
+  - type: notify
+    jira_template: my-jira-template
```

Comment:
we already have a generically used template field, let's not add a new one for the same purpose.

Reply:
I don't understand. Based on what I can see, we have a `template` for email and a `slack_template` for Slack. Are you suggesting that we should reuse the email template for the Jira description? What if the user needs to deliver both email and Jira messages and wants to present the content in different formats?